### PR TITLE
[PF-1394] split predefined and custom config

### PR DESCRIPTION
**Pull request by ktaebum**

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.1.16
+0.1.18
```
Comment (kooyunmo) [inline VERSION:1]: The version is already updated to 0.1.18 in main branch. Please do `git stash` for this file or pull the main branch.

File: pfcli/service/config.py
Comment (kooyunmo) [inline pfcli/service/config.py:1]: At the last week, `pfcli/service/config.py` is separated into multiple files in `pfcli/configurator`. Can you please apply the change you made in this file to https://github.com/friendliai/periflow-cli/blob/main/pfcli/configurator/job.py?

**Review by kooyunmo — CHANGES_REQUESTED** [review]

**Comment by kooyunmo** [comment]
@ktaebum I created a new PR that applies the most up-to-date schema in PR #163. Can I close this PR?

**Comment by ktaebum** [comment]
Close by #163

**Pull request closed without merging**